### DISCUSSION TITLE
Invision Craft icon

### DISCUSSION THREAD

**Discussion by ybandura** [elrumo/macOS_Big_Sur_icons_replacements#1266] [Icon request] [answered]
https://www.invisionapp.com/craft

**Comment by johnlamox** [comment, marked as answer]
Done a quick Icon here. Already submitted it to macosicons.com

![Invision Craft](https://user-images.githubusercontent.com/57749200/127227655-0ebad4d0-4e44-458e-9915-ae2f03f16201.png)

**Reply by ybandura** [reply]
Thank you!!!

**Reply by johnlamox** [reply]
also as icns now here: https://macosicons.com/u/johnlamox
